### DISCUSSION TITLE
how i make custom arrow for opponent? (kade 1.8)

### DISCUSSION THREAD

**Discussion by Benby3** [KadeArchive/Kade-Engine#2905] [Q&A]
i tried this tutorial but doesnt working. so, please, needing help!
https://gamebanana.com/tuts/14274

**Comment by isakube** [comment]
not to be rude but that tutorial is old as fuck, it’s from like 8 months ago i think? you might be able to find something in discussions here if you sort by Show & Tell

**Reply by Benby3** [reply]
nah, dont have any discussion in show & tell teaching how to make it.
